old_theme.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
```

Comment:
We can delete the old theme - no use anymore and if things go wrong, we can always revert commits

Reply:
Cool, will do